aws/rust-runtime/aws-config/src/lib.rs

```diff
@@ -612,6 +614,39 @@ mod loader {
             self
         }
 
+        /// Override the [`StoppedStreamProtectionConfig`] used to build [`SdkConfig`](aws_types::SdkConfig).
```

Comment:
this says `Stopped` should it stay `Stalled`? I assume we want to be consistent

Reply:
Good catch